REGISTRY.md

```diff
@@ -0,0 +1,79 @@
+# LLM Foundry Registry
```

Comment:
why separate file instead of README?

Reply:
the readme gets very long without indirection to sections that might get large. not a super strong opinion.

Reply:
Ok, I think that's fair for now though imo it's a little annoying (I like preview on github)